### ISSUE TITLE
Snyk not able to catch formatting issues in requirements.txt for python dependencies

### ISSUE THREAD

**Issue by pingyangtiaer** [ibm-cloud-docs/ContinuousDelivery#32]
An issue been found that, if there is a formatting issue such as
`cryptography ==3.2`
in which there is a **space** after the package name (however the python pip can install the package as to the version 3.2),
I am wondering Snyk didn't check the whole line but only seeks the space as the end of the line and actually using the latest package instead.
I will suggest in the documentation to include "Correcting the formatting" to make the Snyk work as supposed.
We have done this test on our end.

**Comment by acolleen** [comment]
@pingyangtiaer thank you for your feedback. Can you please indicate which help topic you are referencing so that we can investigate updating  it?

**Comment by pingyangtiaer** [comment]
@acolleen I referring to this [page](https://cloud.ibm.com/docs/ContinuousDelivery?topic=ContinuousDelivery-cd-configure-cra-repos): https://cloud.ibm.com/docs/ContinuousDelivery?topic=ContinuousDelivery-cd-configure-cra-repos, in the section it says Code Risk Analyzer supports the Java™, Node.js, and **Python languages**.
especially for Python | Dependencies are calculated by using the requirements.txt file.
While, we found the requirements.txt need to be strictly formatted, just as we found out: if there is a space " " after the package name, Snyk won't check the version number but instead it assume it uses the latest version. 
So I would suggest to add a format template (or rules) on the requirements.txt.
